base-notebook/Dockerfile.ppc64le

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) IBM Corporation 2016 
```

Comment:
It's fine to have the file in there if it makes things simpler. We can work on it in the future. I think we would need a pretty strong reason to have one file with a different license in the repo, though.